CMakeLists.txt

```diff
@@ -133,7 +133,7 @@ elseif(CPACK_GENERATOR STREQUAL "DEB")
   if (CMAKE_SYSTEM_PROCESSOR STREQUAL "x86_64")
     set(CMAKE_SYSTEM_PROCESSOR "amd64")
   endif()
-  set(IRODS_PACKAGE_FILE_NAME_SUFFIX "_${CPACK_DEBIAN_PACKAGE_VERSION}~${IRODS_LINUX_DISTRIBUTION_VERSION_CODENAME}_${CMAKE_SYSTEM_PROCESSOR}")
+  set(IRODS_PACKAGE_FILE_NAME_SUFFIX "_${CPACK_DEBIAN_PACKAGE_VERSION}-${IRODS_LINUX_DISTRIBUTION_VERSION_CODENAME}_${CMAKE_SYSTEM_PROCESSOR}")
```

Comment:
pretty sure we leave these as tilde as per convention on internet...

Reply:
Originally changed this due to some build or install stage giving a fatal hiccup under some conditions. Will need to investigate